### ISSUE TITLE
Redirect on first login

### ISSUE THREAD

**Issue by brharp** [ccswbs/hjckrrh#452]
The [StartOnline website](http://www.startonline.ca) created in 2015 uses a custom module to redirect new users to a [Qualtrics survey](https://uoguelph.eu.qualtrics.com/SE/?SID=SV_411QtXdSRh79xqZ) the first time they log in. The custom module (so_survey) works like so:

```
function so_survey_user_login (&$edit, $account) {
        /* If this is the first time logging in... */
        if ($account->access == 0) {
                /* ... redirect to the survey. */
                unset($_GET['destination']);
                drupal_goto("https://uoguelph.eu.qualtrics.com/SE/?SID=SV_411QtXdSRh79xqZ");
        }
}
```

**Comment by brharp** [comment]
This can be accomplished without a custom module using the Rules module. Create a rule as follows:

- Name: Survey
- Event: User has logged in
- Conditions: Data value is empty
    - Data selector: site:current-user:last-access
- Actions: Page redirect
    - URL: https://uoguelph.eu.qualtrics.com/SE/?SID=SV_411QtXdSRh79xqZ
    - Force redirect: true
    - Append destination parameter: false

This works because the "last-access" time is not set until *after* the rules run. The first time a user logs in, last-access time will be empty.

Of course, this requires the Rules module be installed and enabled. If there is a way to include this rule in a generic feature (as a "Component" maybe?) then go ahead and do so, otherwise we will need to add something to the documentation on how to set this up.

**Comment by brharp** [comment]
Fixed in 7.4.0
